services/web/server/src/simcore_service_webserver/projects/_comments_handlers.py

```diff
@@ -44,6 +44,8 @@ async def wrapper(request: web.Request) -> web.StreamResponse:
 
         except ProjectNotFoundError as exc:
             raise web.HTTPNotFound(reason=f"{exc}") from exc
+        except ProjectInvalidRightsError as exc:
+            raise web.HTTPUnauthorized(reason=f"{exc}") from exc
```

Comment:
You were responding `web.HTTPForbidden` for the same exception in other handles. Unauthorized i mainly for login/credentials type of errors

Reply:
It's also important to develop a mechanism to include these errors in the OAS. For the moment we are doing it by-hand but we have some ideas on how to automate it. SEE

https://github.com/itisfoundation/osparc-simcore/blob/42cc5e6e0d5fc691bd36a1da5c8a29de6fe50c10/api/specs/web-server/_computations.py#L31-L41

Reply:
Ups you are right, thanks I will fix it.